turbinia/client.py

```diff
@@ -895,6 +904,16 @@ def __init__(self, jobs_blacklist=None, jobs_whitelist=None):
       jobs_whitelist (Optional[list[str]]): The only Jobs we will include to run
     """
     super(TurbiniaCeleryWorker, self).__init__()
+
+    if jobs_whitelist or jobs_blacklist:
```

Comment:
Should we move this into the `DeregisterJobs` function so that we don't have the code duplication here?

Reply:
Done.